solr/core/src/java/org/apache/solr/cli/SolrCLI.java

```diff
@@ -411,21 +411,25 @@ private static void printHelp() {
     print(
         "                                zk ls, zk cp, zk rm , zk mv, zk mkroot, zk upconfig, zk downconfig,");
     print(
-        "                                snapshot-create, snapshot-list, snapshot-delete, snapshot-export, snapshot-prepare-export");
+        "                                snapshot-create, snapshot-list, snapshot-delete, snapshot-export");
     print("");
-    print("  Standalone server example (start Solr running in the background on port 8984):");
+    print("  Start Solr on default port 8983:");
```

Comment:
I think I’m fine with --user-managed being second.  Between the help-text ordering and the “user-managed” name itself (which IMO correctly implies more responsibility+work for the user) - we’re giving new users a pretty substantial nudge towards SolrCloud.  Personally, I think that’s great.

But I'm a little uncomfortable with the wording that's here in the “first spot”.  It doesn't really clarify the mode one way or another.  And that feels a little dangerous in a release that also happens to switch the "default" mode!  90% of users coming from 9.x standalone will read that and (incorrectly) miss that the default mode has changed.  

Please consider adding a tweak here so that the "First spot" does _something_ to indicate the mode ("SolrCloud", mention ZooKeeper, etc.)

Reply:
done